### PR TITLE
Fix spelling mistakes in README

### PR DESCRIPTION
**Pull request by KillianKemps**
Just some changes to make the Readme more readable.

### CODE REVIEW

**Comment by gdchamal** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,24 +4,24 @@
 Status](https://travis-ci.org/CaliOpen/caliopen-dev.svg?branch=master)](https://travis-ci.org/CaliOpen/caliopen-dev)
 
 We use [docker-compose](http://docs.docker.com/compose/) to run [CaliOpen](https://caliopen.org) for
-develoment purpose.
+development purpose.
```
Comment (ziir) [inline README.md]: purposes

File: README.md [hunk 2]
```diff
@@ -4,24 +4,24 @@
...
 
 > Up to now, only external services are running in docker containers.
-> CaliOpen service still run in a virtualenv. This should be fixed shortly.
+> CaliOpen service still runs in a virtualenv, but this should be fixed shortly.
```
Comment (ziir) [inline README.md]: In my opinion, the use of virtualenv is not something that needs fixing. We wish to provide a docker-based development environment, not forbid or discourage the use of virtualenv.

cc @themouette, @gdchamal

File: README.md [hunk 3]
```diff
@@ -4,24 +4,24 @@
...
-While this script is not available, manual installation is required:
+While this script is not available, a manual installation is required:
 
 ``` sh
-# Create CaliOpen workdir
+# Create CaliOpen work directory
 mkdir caliopen && cd $_
 
-# Clone development utilities
+# Clone development utilities in a new bin folder
```
Comment (ziir) [inline README.md]: `bin`

File: README.md [hunk 4]
```diff
@@ -29,37 +29,37 @@ git clone https://github.com/CaliOpen/caliopen-dev.git bin
...
-> cd web
-> git remote add caliopen https://github.com/CaliOpen/caliopen.web.git
-> git remote set-url origin git@github.com:themouette/caliopen.web.git
-> ```
+For instance, to contribute to caliopen.web:
+
+``` sh
+cd web
+git remote add caliopen https://github.com/CaliOpen/caliopen.web.git
+git remote set-url origin git@github.com:themouette/caliopen.web.git
```
Comment (ziir) [inline README.md]: I'd rather have something like `git@github.com:<your username>/caliopen.web.git`

File: README.md [hunk 5]
```diff
@@ -70,16 +70,16 @@ to describe as explicitly as possible what you're trying to achieve.
 
 ## Update Code Base
 
-To update hte whole codebase in bulk, just run `./bin/update`.
+To update the whole codebase in bulk, just run `./bin/update`.
```
Comment (ziir) [inline README.md]: I'd remove `in bulk` from the sentence.

File: README.md (hunk 2 above)
Comment (KillianKemps) [inline README.md]: This is just fixing the way it is written in the README. We can still change our policy concerning virtualenv later.

**Comment by ziir** [comment]
:+1: Thanks Killian

**Comment by esion** [comment]
+1

**Pull request merged**